warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -903,6 +909,18 @@ protected ASTJexlScript processTree(final ASTJexlScript originalQueryTree, Shard
         
         TraceStopwatch stopwatch = null;
         
+        if (!disableWhindexFieldMappings) {
+            stopwatch = timers.newStartedStopwatch("DefaultQueryPlanner - Apply Whindex field mappings");
+            
+            // apply the value-specific field mappings for GeoWave functions
+            queryTree = WhindexVisitor.apply(queryTree, config.getWhindexMappingFields(), config.getWhindexFieldMappings(), metadataHelper);
```

Comment:
Shall we also call this after regex expansion?

Reply:
As in the comment about handling ER nodes, perhaps this is a case we don't care about...